startup/views.py

```diff
@@ -0,0 +1,76 @@
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from .models import startup
+from .serializers import startupSerializer
+#from decorators import ecell_user
+from django.http import HttpResponse
+import csv
+
+
+@api_view(['GET',])
+def get_startups(request, year):
+
+    res_message = ""
+    res_status = ""
+    res_data = []
+
+    startups = startup.objects.filter(year=year, verified=True)
+    if len(startup)>0:
+        res_data    = startupSerializer(startup, many=True,
+                             context={'request':request}).data
+        res_message = "startups Fetched successfully."
+        res_status  = status.HTTP_200_OK
+    else:
+        res_message = "startups Couldn't be fetched"
+        res_status  = status.HTTP_404_NOT_FOUND
+
+    return Response({
+        "message": res_message,
+        "data"   : res_data
+    }, status=res_status)
+
+
+@api_view(['POST',])
+#@ecell_user
+def add_startup(request):
+    if request.ecelluser.user_type in ['GST','VLT','CAB']:
+        return Response({
+            "message":"Unauthorized to view this page"
+        }, status=HTTP_401_UNAUTHORIZED)
+
+    res_message = ""
+    res_data    = []
+
+    startup = startupSerializer(data=request.data)
+
```

Comment:
Use validation for unique constraint to find if the entry already exists. If the entry already exists give user a message stating the same.